src/program/lwaftr/run_nohw/run_nohw.lua

```diff
@@ -17,45 +17,49 @@ local function check(flag, fmt, ...)
 end
 
 local function parse_args(args)
-   local verbosity = 0
+   local opts = {
+      verbosity = false,
+   }
    local conf_file, b4_if, inet_if
-   local bench_file = 'bench.csv'
    local handlers = {
       v = function ()
-         verbosity = verbosity + 1
-      end;
+         opts.verbosity = true
+      end,
       c = function (arg)
          check(file_exists(arg), "no such file '%s'", arg)
          conf_file = arg
-      end;
+      end,
```

Comment:
There was no particular reason for semicolons, other than usually for array-shaped tables I tend to use commas as separators, and for dictionary-shaped tables I tend to use semicolons — that's all. I am aware that the coding style for Snabb is using always commas, so this probably has slipped in unnoticed.